_shows/08_09/a_clockwork_orange.md

```diff
@@ -12,6 +12,10 @@ assets:
   - type: poster
     image: HRfjFr7
 
+cast:
+  - role: Marty and Ensemble
```

Comment:
Should this be `Marty / Ensemble`?

Reply:
Yes, have updated to reflect standard for multi-roling